### ISSUE TITLE
Is there a way to distinguish Open JDK from Oracle JDK when using the Custom JVM option?

### ISSUE THREAD

**Issue by Halahala1993** [threerings/getdown#150]
In our testing we noticed we can't force Getdown to download Oracle JDK 11 if we use Open JDK 11 to initially launch Getdown. It can't differentiate the versions or build numbers between them.

Is there a configuration that could workaround this issue?

**Comment by zilti** [comment]
They aren't meant to be differentiated. They have feature parity, and OpenJDK is the reference implementation.

**Comment by Halahala1993** [comment]
For our use case: on the server side we'd have the Oracle JDK ready to download while OpenJDK is only used to launch Getdown. We want our application to only be launched using an Oracle JDK and to never be launched with OpenJDK

**Comment by zilti** [comment]
This isn't helping with your question, but: when your application only works with one of the two, you should fire your developer.

**Comment by Halahala1993** [comment]
You misunderstand. It's not a constraint on our application we know it'll work on both regardless. 

I'm asking this for the sole purpose of having an answer for customers who only want to run our application on Oracle's JDK.

**Comment by ChristianCiach** [comment]
As a side note: As far as I know there is also no guarantee for different JDK implementations to be compatible in regards to serialization. Today it may be frowned upon, but quite a lot of legacy applications that try to migrate away from Webstart still use ArrayList and other data structures in an RMI-styled manner to communicate between client and server. So, this may be a valid use case to force a specific JDK implementation on the client.

**Comment by sergiorussia** [comment]
@zilti you're wrong when saying "they aren't meant to be differentiated". eg openjdk-8-jre does not include openjfx (JavaFX) on Linux however Oracle JRE does include JavaFX on each platform
